### ISSUE TITLE
[P3.1][Sourcing][Accessibility][p13n dialog] Table column re-ordering info not conveyed to screen reader

### ISSUE THREAD

**Issue by yixuan-liu** [SAP/fundamental-ngx#10872]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
While selecting any table column and perform moving up/down, no info is being conveyed to screen reader. Screen reader need to read out which column is move to which position inside the `Columns` view of p13n dialog.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
- Angular 15
- F-NGX Platform: 0.43.14

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using the `Personalization Dialog` example under https://ng-15-downport--fundamental-ngx.netlify.app/#/platform/table, and perform selection of any table column to move up/down.

![Screenshot 2023-11-01 at 5 21 12 PM](https://github.com/SAP/fundamental-ngx/assets/22454655/9cff0814-289d-47c4-b62a-5bf4e4527670)

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
State, information, properties of all UI elements should be conveyed correctly once to the screen reader users when accessed.

**Comment by nikolay-kolarov** [comment]
Hi @yixuan-liu,

The change happens after the focus position and the moved rows are the ones that were previously selected, so the layout is usable as upon exploring the list the user can understand the change. But I agree that it is far from ideal.

@droshev - Due to such complains for the design of the personalization dialog was changed in UI5: https://ui5.sap.com/#/entity/sap.m.Table/sample/sap.m.sample.p13n.Engine
You can decide how/if/when to plan for such a change, but keep in mind that the current interaction that is implemented is not optimal and you can get further complaints.

Best Regards,
Nikolay

**Comment by droshev** [comment]
@yixuan-liu Nikolay suggested another solution, which requires refactoring and rewriting the component. It will take time and it may bring some breaking changes. The current layout is usable and the screen reader users can explore the list after the changes (it is not ideal from UX point of view). I would suggest we leave the solution as is and wait for future feedback.

**Comment by yixuan-liu** [comment]
Thanks @nikolay-kolarov for the update. We'll inform our testing team about the current resolution and hold on for future fixes when available!
